### ISSUE TITLE
Android14: App crashes with permission errors 

### ISSUE THREAD

**Issue by asherp** [OpenTracksApp/OpenTracks#1749]
### App information
* ID: de.dennisguse.opentracks
* Version: 5491 v4.9.3

### Device information
* Brand: google
* Device: oriole
* Model: Pixel 6
* Id: UP1A.231005.007
* Product: oriole

### Firmware
* SDK: 34
* Release: 14
* Incremental: 2023102300

### Cause of error
```java
Exception in thread "main": java.lang.SecurityException: Starting FGS with type connectedDevice callerApp=ProcessRecord{7c98841 32708:de.dennisguse.opentracks/u0a154} targetSDK=34 requires permissions: all of the permissions allOf=true [android.permission.FOREGROUND_SERVICE_CONNECTED_DEVICE] any of the permissions allOf=false [android.permission.BLUETOOTH_ADVERTISE, android.permission.BLUETOOTH_CONNECT, android.permission.BLUETOOTH_SCAN, android.permission.CHANGE_NETWORK_STATE, android.permission.CHANGE_WIFI_STATE, android.permission.CHANGE_WIFI_MULTICAST_STATE, android.permission.NFC, android.permission.TRANSMIT_IR, android.permission.UWB_RANGING, USB Device, USB Accessory] 
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3066)
	at android.os.Parcel.createException(Parcel.java:3050)
	at android.os.Parcel.readException(Parcel.java:3026)
	at android.os.Parcel.readException(Parcel.java:2968)
	at android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:6776)
	at android.app.Servic

**Comment by asherp** [comment]
I was able to get it working by

1. Exporting tracks
2. Uninstall app
3. Reinstall app
4. Allow permissions when prompted

**Comment by dennisguse** [comment]
Mhh.
That's report number three. 

Can you try to reproduce this?
My guess is that the Bluetooth permission now implicitly became mandatory. 
Can you check if you remove this in Android settings for OpenTracks (ask or deny). 
... And see what happens.

**Comment by fabibau** [comment]
Same issue here. After allowing nearby devices in settings it worked again.

**Comment by dennisguse** [comment]
@fabibau can you make a screenshot of the UI where you allowed nearby devices?
I only have an Android 13 at hand...

**Comment by fabibau** [comment]
![Screenshot_20231028-192956](https://github.com/OpenTracksApp/OpenTracks/assets/132608061/2ecc1e8d-02d3-467c-a05f-a187d827cbda)
It is the second permission "Geräte in der Nähe".

**Comment by dennisguse** [comment]
I guess, I understood the problem: Bluetooth permissions are now always mandatory. The same applies to GPS. 

OpenTracks cannot start it's foreground service without this anymore. Until Android 13 those permissions are optional.

**Comment by dennisguse** [comment]
@fabibau @asherp Could you test this? See #1750 

@pstorch you as well?

**Comment by pstorch** [comment]
I'm still on Android 13.

**Comment by fabibau** [comment]
@dennisguse With the nightly build from your PR the app does not crash any longer if you deny the nearby devices permission. It also asks for granting it at startup and seems to work properly.

**Comment by dennisguse** [comment]
@fabibau thanks for testing; released as v4.9.5
